doc/source/whatsnew/v0.18.2.txt

```diff
@@ -76,6 +76,7 @@ Other enhancements
 
 - The ``pd.read_csv()`` with ``engine='python'`` has gained support for the ``decimal`` option (:issue:`12933`)
 - The ``pd.read_csv()`` with ``engine='python'`` has gained support for the ``na_filter`` option (:issue:`13321`)
+- The ``pd.read_csv()`` with ``engine='python'`` has gained support for the ``compact_ints`` and ``use_unsigned`` options (:issue:`13323`)
```

Comment:
That is a fair point, but I did add support though.  The exact direction of this PR is still somewhat in flux, and I'll adjust the `whatsnew` once things have settled down a bit.  More importantly, I would like to figure out why Travis is unhappy with me at the moment.